pontoon/administration/templates/admin_project.html

```diff
@@ -80,6 +80,12 @@ <h1 id="title">
           {{ repo_form.DELETE }}
         </div>
       </section>
+      <section class="branch-wrapper">
+        <div class="branch">
+          {{ repo_form.branch.label_tag() }}
+          {{ repo_form.branch }}
+        </div>
+      </section>
```

Comment:
We need to add a similar block also to the `.repository-empty` block below, which is used as a template for adding new repositories.